modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/praetorian.dm

```diff
@@ -0,0 +1,122 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat/praetorian
+	name = "alien praetorian"
+	desc = "An alien that looks like the awkward half-way point between a queen and a drone, in fact that's likely what it is."
+	caste = "praetorian"
+	maxHealth = 400
+	health = 400
+	icon_state = "alienpraetorian"
+	/// Holds the improved healing aura ability to be granted to the praetorian later
+	var/datum/action/cooldown/alien/skyrat/heal_aura/juiced/heal_aura_ability
+	/// Holds the less lethal tail sweep ability to be granted to the praetorian later
+	var/datum/action/cooldown/spell/aoe/repulse/xeno/skyrat_tailsweep/hard_throwing/tail_sweep
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	next_evolution = /mob/living/carbon/alien/humanoid/skyrat/queen
+
+/mob/living/carbon/alien/humanoid/skyrat/praetorian/Initialize(mapload)
+	. = ..()
+	heal_aura_ability = new /datum/action/cooldown/alien/skyrat/heal_aura/juiced()
+	heal_aura_ability.Grant(src)
+
+	tail_sweep = new /datum/action/cooldown/spell/aoe/repulse/xeno/skyrat_tailsweep/hard_throwing()
+	tail_sweep.Grant(src)
+
+	REMOVE_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+
+	add_movespeed_modifier(/datum/movespeed_modifier/alien_big)
+
+/mob/living/carbon/alien/humanoid/skyrat/praetorian/Destroy()
+	QDEL_NULL(heal_aura_ability)
+	return ..()
+
+/mob/living/carbon/alien/humanoid/skyrat/praetorian/create_internal_organs()
+	internal_organs += new /obj/item/organ/internal/alien/plasmavessel/large
+	internal_organs += new /obj/item/organ/internal/alien/neurotoxin/spitter
+	internal_organs += new /obj/item/organ/internal/alien/resinspinner
+	..()
+
+/datum/action/cooldown/alien/skyrat/heal_aura/juiced
+	name = "Strong Healing Aura"
+	desc = "Friendly xenomorphs in a longer range around yourself will receive passive healing."
+	button_icon_state = "healaura_juiced"
+	plasma_cost = 100
+	cooldown_time = 90 SECONDS
```

Comment:
I haven't seen this one in play, but passive healing mobs (with a fair bit of sustain) shouldn't also have two different variants with the ability to give even more passive healing, or atleast, not for as long as it lasts